json-endpoints/jetpack/class.jetpack-json-api-themes-install-endpoint.php

```diff
@@ -14,6 +14,12 @@ protected function install() {
 
 		foreach ( $this->themes as $theme ) {
 
+			// hook to allow alternative install method
+			if ( wp_endswith( $theme, '-wpcom' )
```

Comment:
Yeah I'm fine with removing the condition. I think only put it there to make the example plugin simpler.